test/commons/dom/visually-contains.js

```diff
@@ -47,4 +47,12 @@ describe('dom.visuallyContains', function () {
 		assert.isTrue(axe.commons.dom.visuallyContains(target, target.parentNode));
 	});
 
+	it('should return true when element is inline', function () {
```

Comment:
Also: this is just for the `visuallyContains` method, which checks the element's bounding box against the parent bounding box and parent scrollArea: it doesn't work on inline elements at all. The scroll area is completely off cross-platform so it won't return consistently.